beast/observationmodel/observations.py

```diff
@@ -90,30 +83,11 @@ def __getitem__(self, *args, **kwargs):
 
     def keys(self):
         """ Returns dataset content names """
-        return self.data.keys()
+        return list(self.data.keys())
```

Comment:
We should respect the way python works and therefore returns the keys not the list of them.

Reply:
Fixed.  list() removed.